src/include/block.h

```diff
@@ -223,6 +223,7 @@ struct __wt_bm {
     int (*write)(
       WT_BM *, WT_SESSION_IMPL *, WT_ITEM *, WT_PAGE_BLOCK_META *, uint8_t *, size_t *, bool, bool);
     int (*write_size)(WT_BM *, WT_SESSION_IMPL *, size_t *);
+    size_t (*encrypt_skip)(WT_BM *, WT_SESSION_IMPL *, bool);
```

Comment:
I'm sad that we need a function like this, but understand why.  I think once we get rid of the pantry block manager and we put the block header first, then we can remove this custom `encrypt_skip` function.

Reply:
Added a comment in ab8121d7b9ccd38510fb1b627f705cf30ab03f72 as a reminder to look at removing it.

Reply:
I don't think we can get rid of it because it skips both the page header and the block header. Even we change the sequence, the size will still be different.

Reply:
Right, but the block header contains a `header_size` which we can populate differently based on whether it's a delta.

Reply:
OK, I don't know that. I somehow get why we put the page header before the block header. I feel this is mainly due to convenience reasons. At the btree level, we only work with the page header so it is more convenient to have it at front. We can simply cast the page image pointer to access it.